demo/components/page-transition/PageTransition.module.css

```diff
@@ -0,0 +1,118 @@
+.fade {
+    will-change: opacity;
```

Comment:
This does not force this element into a composite layer, nor it improves performance in recent browsers. Don't think we need it 😄

Reply:
But it's a demo, so I guess you can leave it 🤷‍♂

Reply:
Yep I've removed it!